config/projects-to-test/openjdk20-excluded.files

```diff
@@ -0,0 +1,543 @@
+<!-- This file was generated by script at https://github.com/checkstyle/checkstyle/issues/13402#issue-1806844349 -->
+<!-- Excluded due to exception; issues opened -->
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8654-->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]jdk[\\/]javadoc[\\/]doclet[\\/]testSupplementary[\\/]C.java$"/>
+</module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]hotspot[\\/]jtreg[\\/]runtime[\\/]exceptionMsgs[\\/]methodPrinting[\\/]TestPrintingMethods.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8650 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]MethodParameters[\\/]UncommonParamNames.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8655 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]UnicodeAtEOL.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8656 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]UnicodeCommentDelimiter.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8654 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]FirstChar2.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/13279 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]patterns[\\/]ForEachPatterns.java$"/>
+</module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]patterns[\\/]GenericRecordDeconstructionPattern.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/13411 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]SwitchScope.java$"/>
+</module>
```

Comment:
This is a "new" input (doesn't exist in JDK < 20), but doesn't involve new syntax. Issue has been opened.